app/views/disassociate_ucr.scala.html

```diff
@@ -54,13 +53,13 @@
                     value = Some(DisassociateKind.Ducr.formValue),
                     content = Text(messages("disassociate.ucr.ducr")),
                     checked = form("kind").value.contains(DisassociateKind.Ducr.formValue),
-                    conditionalHtml = Some(exportsInputText(form("ducr")))
+                    conditionalHtml = Some(exportsInputText(form("ducr"), "mucrOptions.ducr.label"))
```

Comment:
Should these be using the same message key for all forms?  If they are really supposed to be the same text then maybe some global/site labels?

Reply:
Good point.
I have gone with 
```
site.inputText.ducr.label = Enter an existing DUCR
site.inputText.mucr.label = Enter an existing MUCR
site.inputText.newMucr.label = Enter a new MUCR
site.inputText.newMucr.label.hint = For example ‘GB/123456789100-AB123’
```